packages/tools/cookbook/README.md

```diff
@@ -0,0 +1,56 @@
+# Kadena.js Cookbook
+
+This project demonstrates common use cases for `@kadena/client` and
+`@kadena/pact-cli` for _smart contracts_
+
+<p align="center">
+
+  <picture>
+
+    <source srcset="https://github.com/kadena-community/kadena.js/raw/master/common/images/Kadena.JS_logo-white.png" media="(prefers-color-scheme: dark)"/>
+
+    <img src="https://github.com/kadena-community/kadena.js/raw/master/common/images/Kadena.JS_logo-black.png" width="200" alt="kadena.js logo" />
+
+  </picture>
+
+</p>
```

Comment:
This part should be an image but on preview it's shown as a codeblock. Maybe autoformatting is breaking this?